modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/util/TaskContext.java

```diff
@@ -135,6 +172,20 @@ private DataSourceObject getDataSourceObject(String objectId) {
         return null;
     }
 
+    public Set<PostActionParameterObject> getPostActionParameters() {
+        return postActionParameters;
+    }
+
+    private PostActionParameterObject getPostActionParameter(String objectId, String objectKey) {
+        for (PostActionParameterObject postActionParameterObject : postActionParameters) {
+            if (postActionParameterObject.getObjectId().equals(objectId) &&
```

Comment:
getObjectId and getObjectValue can be null ? if yes please adjust this conditions